### ISSUE TITLE
Support building tests in parallel (at least for Nightly Test)

### ISSUE THREAD

**Issue by kwk** [kwk/test-llvm-bz-import-5#4320]
This issue was imported from Bugzilla https://bugs.llvm.org/show_bug.cgi?id=4320.

**Comment by kwk** [comment]
Closing issue because of bugzilla's bug state (RESOLVED) and resolution (WONTFIX).
